GruntFile.js

```diff
@@ -18,8 +24,60 @@ module.exports = function(grunt) {
 			}
 		},
 
-		'clean': {
-			css: ["dist/*.css"]
+		'babel': {
+			options: {
+				presets: ['es2015']
+			},
+			test: {
+				files: {
+					'test/build/lib.js': 'src/js/lib.es6.js',
+					'test/build/DummySpec.js': 'test/js/spec/DummySpec.es6.js'
+				}
+			},
+			dist: {
+				files: {
+					'dist/lib.js': 'src/js/lib.es6.js'
+				}
+			}
+
+		},
+
+		'webpack': {
+			test: {
+				entry: ['./' + TEST_DEST + 'lib.js', './'+ TEST_DEST + '/DummySpec.js'],
```

Comment:
I wanted to have two entry points here, and use them below as jasmine src and specs properties

like 
``` 'webpack': {
  test: {
  entry: {
     lib: './' + TEST_DEST + 'lib.js',
  spec: './'+ TEST_DEST + '/DummySpec.js'
},
output: {
 filename: '${name}.compiled.js'
}
``` 
I did get two files and they were included in the jasmine test runner, however they didn't work together.